_content/papers/wonhyo_muller-a-charles.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Wonhyo"
+authors:
+  - "A. Charles Muller"
+external_url: "https://web.archive.org/web/20241130202048if_/http://www.acmuller.net/kor-bud/2015-11-wonhyo-buddhist-world.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1e0-bR5Ge-oanOn9_Wm0vx1mpgfMz4Qyt/view?usp=sharing"
+course: korean
+tags:
+  - medieval
```

Comment:
`medieval` is for things of interest to both Mahāyāna and Theravāda scholars. Let's go with `east-asian-roots` instead?